src/otx/core/types/task.py

```diff
@@ -15,7 +15,8 @@ class OTXTaskType(str, Enum):
     ACTION_CLASSIFICATION = "ACTION_CLASSIFICATION"
 
     # Anomaly Detection
-    ANOMALY_CLASSIFICATION = "ANOMALY_CLASSIFICATION"
```

Comment:
As I know, both ANOMALY_CLASSIFICATION and ANOMALY should exist in phase 1 of the anomaly task reduction. Please don't delete this for Geti.

Reply:
This comment is for OTX 1.x only.